monitoring/v3/src/test/java/com/example/monitoring/AlertIT.java

```diff
@@ -66,7 +66,7 @@ public void tearDown() {
   @Test
   public void testListPolicies() throws IOException {
     AlertSample.main(new String[] {"list"});
-    assertTrue(bout.toString().contains(testPolicyName));
+    assertTrue(bout.toString().contains("My Uptime Check Policy"));
```

Comment:
Thanks for this fix @anguillanneuf! Should we consider referencing the updated variable `testPolicyName` in each test, rather than repeating the raw string?

Reply:
ohh, this wasn't the root of the problem as i later found out.